src/runtime_src/core/include/experimental/xrt_error.h

```diff
@@ -105,4 +105,22 @@ xrtErrorGetLast(xrtDeviceHandle handle, xrtErrorClass ecl, xrtErrorCode* error,
 int
 xrtErrorGetString(xrtDeviceHandle, xrtErrorCode error, char* out, size_t len, size_t* out_len);
 
+/**
+ * xrtErrorGetJson - Get the description string in json format of a given error code.
+ *
+ * @handle:       Device handle.
+ * @error:        XRT error code.
+ * @out:          Preallocated output buffer for the error string.
+ * @len:          Length of output buffer.
+ * @out_len:      Output of length of message, ignored if null.
+ *
+ * Return:        0 on success or appropriate XRT error code.
+ *
+ * Specifying out_len while passing nullptr for output buffer will
+ * return the message length, which can then be used to allocate the
+ * output buffer itself.
+ */
+int
+xrtErrorGetJson(xrtDeviceHandle, xrtErrorCode error, char* out, size_t len, size_t* out_len);
```

Comment:
Not needed....see comments above.

Reply:
done

Reply:
Remove the function declaration if it should not be exposed to end user.  The definition was removed from xrt_error.cpp so maybe this is an oversight?